launch/docker_test.go

```diff
@@ -129,6 +164,58 @@ func TestRunBuild(t *testing.T) {
 	}
 }
 
+func TestRunBuildWithSudo(t *testing.T) {
+	defer func() {
+		execCommand = exec.Command
+	}()
+
+	d := &docker{
+		volume:            "SD_LAUNCH_BIN",
+		setupImage:        "launcher",
+		setupImageVersion: "latest",
+		useSudo:           true,
+	}
+
+	testCase := []struct {
+		name             string
+		id               string
+		expectError      error
+		expectedCommands []string
+		buildConfig      buildConfig
+	}{
+		{"success", "SUCCESS_RUN_BUILD_SUDO", nil,
+			[]string{
+				"docker pull node:12",
+				fmt.Sprintf("docker container run --rm -v /:/sd/workspace/src/screwdriver.cd/sd-local/local-build -v sd-artifacts/:/test/artifacts -v %s:/opt/sd node:12 /opt/sd/local_run.sh ", d.volume)},
```

Comment:
I think `sudo` command is not included in expectedCommands, but is it OK?

Reply:
I fixed.